workdir/test-list.sh

```diff
@@ -0,0 +1,21 @@
+# List of regex that will match the tests you want to run
+# NOTE: This includes both build and run tests
+# e.g. test_include_list=("gcc-.*-initrd-pie-.*" "clang-.*-initrd.sh|9pfs.sh")
+test_include_list=()
+
+# List of regex that will match the tests that should be skipped
+# NOTE: This includes both build and run tests
+# e.g. test_exclude_list=("clang-.*-pie.*" ".*-fc-aarch64.*")
+test_exclude_list=()
+
+# List of regex that will match the tests you want to run
+# NOTE: This includes ONLY run tests. No not include the compiler name in the regex,
+# as the run scripts names do not include it.
+# e.g. run_test_include_list=(".*x86_64-initrd.*" ".*-fc-aarch64.*")
+run_test_include_list=()
+
+# List of regex that will match the tests that should be skipped
+# NOTE: This includes ONLY run tests. No not include the compiler name in the regex,
+# as the run scripts names do not include it.
+# e.g. run_test_exclude_list=(".*x86_64-initrd.*" ".*-fc-aarch64.*")
+run_test_exclude_list=()
```

Comment:
This will not work. I want to skip running of images that were generated using PIE. The `-pie` suffix is only part of the build script name, not the running script.

I would use the same type of regexes for `run_test_include_list` and `run_test_exclude_list`.

The match is done on the build script as well. But, in case a match is found, there is no running of the run step. Only the build step remains.